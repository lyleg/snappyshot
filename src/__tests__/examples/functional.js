import {add} from '../../examples/functional.js'
  it('default snapshot for add', () => {
    expect(add(2000,2000)).toMatchSnapshot()
  });