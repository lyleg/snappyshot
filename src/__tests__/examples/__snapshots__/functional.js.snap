exports[`test default snapshot for add 1`] = `4000`;
